src/ocamlorg_frontend/layouts/learn_layout.eml

```diff
@@ -1,121 +1,107 @@
+type active_icon_link = 
+  | Learn
+  | Exercises
 
-let render
-?use_swiper
-?current_tutorial
-?styles
+let sidebar_icon_link_block
+~current =
+  <div class="space-y-4">
+    <a href="<%s Url.learn %>" class="flex justify-start items-center">
+      <div class="bg-primary-600 text-white rounded w-8 h-8 flex items-center justify-center">
+        <svg xmlns="http://www.w3.org/2000/svg" class="h-5 w-5" fill="none" viewBox="0 0 24 24"
+          stroke="currentColor">
+          <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2"
+            d="M12 6.253v13m0-13C10.832 5.477 9.246 5 7.5 5S4.168 5.477 3 6.253v13C4.168 18.477 5.754 18 7.5 18s3.332.477 4.5 1.253m0-13C13.168 5.477 14.754 5 16.5 5c1.747 0 3.332.477 4.5 1.253v13C19.832 18.477 18.247 18 16.5 18c-1.746 0-3.332.477-4.5 1.253" />
+        </svg>
+      </div>
+      <div class="ml-3 font-semibold <%s if current != Learn then "opacity-70" else "" %>">Learn OCaml</div>
```

Comment:
I find documentation a bit better: Learn is the name of the entire section, including the exercises, manual, API reference, etc.

Reply:
I was seriously confused that "Documentation" would link to the "Learn" page of the top navbar. There is no recognizability between "Documentation" and "Learn".

Reply:
Ok, let's go with Learn then 🙂